src/ResourceManager/DataLakeStore/Commands.DataLakeStore/DataPlaneCommands/ExportAzureRmDataLakeStoreChildItemProperties.cs

```diff
@@ -0,0 +1,97 @@
+                "In the format '/folder/file.txt', " +
+                "where the first '/' after the DNS indicates the root of the file system.")]
+        [ValidateNotNull]
+        public DataLakeStorePathInstance Path { get; set; }
+
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 2, Mandatory = true,
+            HelpMessage = "Path to output file. Can be a Local path or Adl Path. By default"+
+                          " it is local. If SaveToAdl is pecified then it is an ADL path in the same account")]
+        [ValidateNotNullOrEmpty]
+        public string OutputPath { get; set; }
+
+        [Parameter(ValueFromPipelineByPropertyName = true, Mandatory = false,
+            HelpMessage =
+                "Save output to ADL Store, in the same account. The OutputPath parameter should be the full ADL path to output to." + 
+                "Default is to save to a local file. In that case, OutputPath specifies the pathto local file.")]
+        public SwitchParameter SaveToAdl { get; set; }
+
+        [Parameter(ValueFromPipelineByPropertyName = true, Mandatory = false,
+            HelpMessage =
+                "Show stats at file level (default is to show directory-level info only)")]
+        public SwitchParameter IncludeFile { get; set; }
+
+        [Parameter(ValueFromPipelineByPropertyName = true, Mandatory = false,
+            HelpMessage = "Maximum depth from the Path specified, to which disk usage or acl is displayed")]
+        public int MaximumDepth { get; set; } = int.MaxValue;
+
+        [Parameter(ValueFromPipelineByPropertyName = true, Mandatory = false,
+            HelpMessage =
+                "Number of files/directories processed in parallel. Optional: a reasonable default will be selected")]
+        public int Concurrency { get; set; } = -1;
+
+        [Parameter(ValueFromPipelineByPropertyName = true, ParameterSetName = BaseParameterSetName, Mandatory = true,
+            HelpMessage = "Retrieves the disk usage starting from the Path specified")]
+        [Parameter(ValueFromPipelineByPropertyName = true, ParameterSetName = GetBothParameterSetName, Mandatory = true,
+            HelpMessage = "Retrieves the disk usage starting from the Path specified")]
+        public SwitchParameter GetDiskUsage { get; set; }
+
+        [Parameter(ValueFromPipelineByPropertyName = true, ParameterSetName = GetAclParameterSetName, Mandatory = true,
+            HelpMessage = "Retrieves the acl starting from the Path specified")]
+        [Parameter(ValueFromPipelineByPropertyName = true, ParameterSetName = GetBothParameterSetName, Mandatory = true,
+            HelpMessage = "Retrieves the acl starting from the Path specified")]
+        public SwitchParameter GetAcl { get; set; }
+
+        [Parameter(ValueFromPipelineByPropertyName = true, ParameterSetName = GetAclParameterSetName, Mandatory = false,
```

Comment:
Why are these ValueFromPipelineByPropertyName? please remove

Reply:
@markcowl Sorry I do not understand. Based on all datalake cmdlets I have seen, all the existing parameter is "ValueFromPipelineByPropertyName = true". Thats why I made it same for these. So why do we want them to be false for this cmdlet? Is there a particular reason why we do not want to allows us to send a value to these parameters from pipeline input?